### DISCUSSION TITLE
EPSTFL=1 error in fortran

### DISCUSSION THREAD

**Discussion by EGSnoob32** [nrc-cnrc/EGSnrc#868] [Questions] [answered]
Hi all, I am an EGSNRC BEAMNRC noob. I have working through the tutes to get better but I am definitely stuck

This is my example file p4icomp.pegs4inp file which compiles fine without the EPSTFL=1, currently I am getting a fortran error At line 1978 of file pegs4_linux64.f (unit = 20, file = 'fort.20')

I am running the command as p4 -i p4icomp [-d sodiumchloride], anyone got any ideas? am I putting EPSTFL in the wrong place? I have made sure to have sodiumchloride.density in the right file location /density_corrections/compounds

COMP
 &INP NE=2,RHO=2.1650000,PZ(1)=1,PZ(2)=1,EPSTFL=1 &END
![Input](https://user-images.githubusercontent.com/104816570/166408841-1923013d-6a7c-4833-a5aa-e478aa0dd0cf.PNG)
![Density file](https://user-images.githubusercontent.com/104816570/166408843-4d620abc-f1cc-43fa-8015-170b11b02a40.PNG)
![Command line](https://user-images.githubusercontent.com/104816570/166408846-a32abca5-7c7c-477c-b674-99d03d0e8f14.PNG)

SODIUM CHLORIDE     NACL
NA CL
ENER
 &INP AE=0.521,UE=50.511,AP=0.01,UP=50 &END
TEST
 &INP  &END
PWLF
 &INP  &END
DECK
 &INP  &END

**Comment by rtownson** [comment, marked as answer]
This input file worked for me! You just need to remove the brackets from the command when you type it. The `[ ]` are meant to indicate optional arguments. You don't actually include them. So it should look like this:

```
p4 -i p4icomp -d sodiumchloride
```

(Question also posted [here](https://www.reddit.com/r/EGSnrc/comments/uh9572/esptfl1_error/))

**Reply by EGSnoob32** [reply]
Thank you!
